PanKanapka/clients/views.py

```diff
@@ -1,14 +1,34 @@
-from django.http import HttpResponseRedirect
+from django.http import HttpResponseRedirect,HttpResponse
 from django.contrib.auth import logout
 from django.contrib import messages
 from django.contrib.auth.decorators import login_required
+from django.shortcuts import render, redirect
+from flask import render_template
```

Comment:
Chyba nie jest dobrym pomysłem mieszkanie django z flaskiem. To są dwa różne frameworki o podobnej funkcjonalności (django jest bardziej kombajnem).
Internety mówią, że może to powodować później problemy (nie wiem dokładnie jakie):
_If you are using this two in one application then there a lot of issues to think about bridging between the two apps, like if a user is logged in or not._

Reply:
Tak, racja, 
pewnie jakiś błąd przy kopiowaniu, bo to i tak nigdzie nie jest użyte